_pages/team.html

```diff
@@ -69,73 +79,63 @@ <h5>Stephanie Synnott</h5>
         <div>
-            <img src="/assets/img/team/ben-updike.png">
-            <h5>Ben Updike</h5>
-            <p>Ben is a mechanic.</p>
+            <img src="/assets/img/team/mira-vega.png">
+            <h5>Mira Vega</h5>
+            <p>Mira is a mechanic.</p>
         </div>
         <div>
-            <img src="/assets/img/team/anuragi-thapliya.png">
-            <h5>Anuragi Thapliya</h5>
-            <p>Anuragi is a mechanic.</p>
+            <img src="/assets/img/team/nikita-solovyov.png">
+            <h5>Nikita Solovyov</h5>
+            <p>Nikita works on scouting analysis.</p>
         </div>
         <div>
-            <img src="/assets/img/team/carolyn-baek.png">
-            <h5>Carolyn Baek</h5>
-            <p>Carolyn is a mechanic. Her Korean name means "pine tree."</p>
+          <img src="/assets/img/team/ian-zullo.png">
+          <h5>Ian Zullo</h5>
+          <p>Ian is a graphic designer, programmer, and outreach member. He creates various logo designs and concepts, and also works on the website with Andrew and Erik.</p>
         </div>
         <div>
             <img src="/assets/img/team/ethan-sisbarro.png">
             <h5>Ethan Sisbarro</h5>
             <p>Ethan works on game strategy.</p>
         </div>
+        <div>
+            <img src="/assets/img/team/john-grondelski.png">
+            <h5>John Grondelski</h5>
+            <p>John works on outreach. He has been a crucial help in our pursuit of the Chairman's award in past years.</p>
+        </div>
         <div>
             <img src="/assets/img/team/cole-tarter.png">
             <h5>Cole Tarter</h5>
             <p>Cole is a mechanic.</p>
         </div>
         <div>
-            <img src="/assets/img/team/jakub-leski.png">
-            <h5>Jakub Leski</h5>
-            <p>Jakub is a mechanic.</p>
-        </div>
-        <div>
-            <img src="/assets/img/team/mira-vega.png">
-            <h5>Mira Vega</h5>
-            <p>Mira is a mechanic.</p>
-        </div>
-        <div>
-            <img src="/assets/img/team/keenan-hom.png">
-            <h5>Keenan Hom</h5>
-            <p>Keenan is a mechanic.</p>
+            <img src="/assets/img/team/carolyn-baek.png">
+            <h5>Carolyn Baek</h5>
+            <p>Carolyn is a mechanic. Her Korean name means "pine tree."</p>
         </div>
         <div>
-            <img src="/assets/img/team/jack-evans.png">
-            <h5>Jack Evans</h5>
-            <p>Jack is a mechanic.</p>
+          <img src="/assets/img/team/fernanda-molina.png">
+          <h5>Fernanda Molina</h5>
+          <p>Fernanda is a programmer with experience in Java and Python. She works on helping fix issues with the robot code and updating the team's website.</p>
         </div>
         <div>
-            <img src="/assets/img/team/nikita-solovyov.png">
-            <h5>Nikita Solovyov</h5>
-            <p>Nikita works on scouting analysis.</p>
+            <img src="/assets/img/team/jakub-leski.png">
```

Comment:
Honestly, remove everyone with <100 hours if you like. Do it in a separate commit though.

Reply:
On the same pull?

Reply:
Oof. Maybe put the cutoff at 75?

Reply:
Sure, 75 is okay.

Reply:
On second thought, let's not make cuts on the website until we know more about who will be on next year's team.